libs/ui-components/src/icon/icon.config.ts

```diff
@@ -22,6 +22,7 @@ const icons = [
   'category-news',
   'category-reports',
   'category-search',
+  'check',
```

Comment:
There is already an existing icon, it is called "tick" could you use it and remove the new .svg ?
If you find this name better, please rename the previous one then